### PR TITLE
TRUNK-5343: Add existingDiagnosis property to diagnosis class.

### PR DESCRIPTION
**Pull request by pluwum**
## Description of what I changed
The PR adds existingDiagnosis property to diagnosis class.

## Issue I worked on
see https://issues.openmrs.org/browse/TRUNK-5343

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

- [ ] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/emrapi/diagnosis/Diagnosis.java [hunk 1]
```diff
@@ -36,6 +36,11 @@
 
     Obs existingObs;
 
+    /**
+     * @since 1.25.0
+     */
+    Diagnosis existingDiagnosis;
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/module/emrapi/diagnosis/Diagnosis.java]: Make this an Integer

**Comment by dkayiwa** [comment]
Are these the only changes that you made?

**Comment by pluwum** [comment]
@dkayiwa , for this module, yes.

File: api/src/main/java/org/openmrs/module/emrapi/diagnosis/Diagnosis.java (hunk 1 above)
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/module/emrapi/diagnosis/Diagnosis.java]: Make this of type integer.

Comment (pluwum) [reply api/src/main/java/org/openmrs/module/emrapi/diagnosis/Diagnosis.java]: @dkayiwa Done.

File: api/src/main/java/org/openmrs/module/emrapi/diagnosis/Diagnosis.java [hunk 2]
```diff
@@ -36,6 +36,11 @@
 
     Obs existingObs;
 
+    /**
+     * @since 1.25.0
+     */
+    Integer existingDiagnosis;
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/module/emrapi/diagnosis/Diagnosis.java]: Can we rename this to existingDiagnosisId

Comment (dkayiwa) [reply api/src/main/java/org/openmrs/module/emrapi/diagnosis/Diagnosis.java]: You know what, just leave it at this for now.

**Pull request merged**